spec/requests/schemes_controller_spec.rb

```diff
@@ -586,13 +600,18 @@
         end
       end
 
-      context "when required organisation id param is missing" do
-        let(:params) { { "scheme" => { "service_name" => "qweqwer", "sensitive" => "Yes", "owning_organisation_id" => "", "scheme_type" => "Foyer", "registered_under_care_act" => "Yes – part registered as a care home" } } }
+      context "when organisation id param refers to a non-stock-owning organisation" do
+        let(:organisation_which_does_not_own_stock) { FactoryBot.create(:organisation, holds_own_stock: false) }
+        let(:params) do
+          { scheme: {
+            owning_organisation_id: organisation_which_does_not_own_stock.id,
```

Comment:
@jamesrose changing this to `owning_organisation: organisation_which_does_not_own_stock` gives the following error:

`Failure/Error: unless owning_organisation.holds_own_stock?
     
     NoMethodError:
       undefined method `holds_own_stock?' for nil:NilClass
     
           unless owning_organisation.holds_own_stock?
                                     ^^^^^^^^^^^^^^^^^
     # ./app/models/scheme.rb:217:in `validate_owning_organisation'
     # ./app/controllers/schemes_controller.rb:86:in `create'
     # ./spec/requests/schemes_controller_spec.rb:612:in `block (5 levels) in <top (required)>'`

Reply:
Ah, apologies. This is `params` which isn't applicable. Please ignore :)